### PR TITLE
SUL23-60 Style the header and footer components

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Styles and interaction for the main menu
- Styles for the secondary navigation on the left sidebar.

# Steps to Test

1. view the preview link from netlify
2. interact with the main menu in mobile and on desktop
3. verify you can access everything and it function as you would expect it to.

<a href="https://gitpod.io/#https://github.com/SU-SWS/sulgryphon-nextjs/pull/30"></a>

### CODE REVIEW

File: components/patterns/lockup.tsx [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
+    <div {...props}>
+      <Link href="/" className="su-no-underline">
+        <div className="su-flex md:su-flex-row su-basefont-19">
+          <Logo/>
+          <div className="lg:su-flex">
+            <div
+              className="lg:su-h-[45px] su-logo su-text-cardinal-red su-type-4 su-leading-[initial] su-pr-5 su-mr-5 lg:su-border-r su-border-solid su-border-black">
+              Stanford
+            </div>
+            <div className="su-text-20 lg:su-text-m2 su-font-regular su-text-black su-flex su-items-end">
```
Comment (jdwjdwjdw) [inline components/patterns/lockup.tsx:16]: Can you make this uppercase to match figma?

File: components/patterns/lockup.tsx [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
-          <div
-            className=" su-text-25 md:su-text-m2 su--mt-3 md:su-mt-0 su--ml-2 md:su-ml-0 su-font-regular su-relative su-top-6 su-text-black ">
-            {process.env.NEXT_PUBLIC_SITE_NAME}
+    <div {...props}>
+      <Link href="/" className="su-no-underline">
+        <div className="su-flex md:su-flex-row su-basefont-19">
+          <Logo/>
+          <div className="lg:su-flex">
+            <div
+              className="lg:su-h-[45px] su-logo su-text-cardinal-red su-type-4 su-leading-[initial] su-pr-5 su-mr-5 lg:su-border-r su-border-solid su-border-black">
```
Comment (jdwjdwjdw) [inline components/patterns/lockup.tsx:13]: Looks like the recent [decanter font addition](https://github.com/SU-SWS/sulgryphon-nextjs/pull/24/files#diff-4ee0e71a2145586038d583d58bcbd7aaa6b42318f417f5b62f2cbee48931861b) are causing some adjustment issues that will need to be fixed. And can we add some more spacing between the logo and `Stanford`?
Screen Shot 2022-11-21 at 12 01 53 PM

**Review by jdwjdwjdw — CHANGES_REQUESTED** [review]
Added some notes - looks great overall. Here a few additional questions:
- Why is the `Placeholder no link` dropdown higher than the others/have a font-weight of 700 vs 600 of the others?
Screen Shot 2022-11-21 at 11 11 31 AM
- I'm Currently unable to see / access dropdown items beyond level 2 if it's the last menu item (getting cut off - )
image
Screen Shot 2022-11-21 at 11 18 59 AM
- Are we not worrying about the search part right now?
Screen Shot 2022-11-21 at 12 35 27 PM
- Looks like figma is showing a redline on hover for dropdown icons, and sub-menu links
image
image

**Comment by pookmish** [comment]
I've adjusted some of your feedback and also implemented some changes after review with Ishi. Gonna merge this as we'll continue some iterative improvements on the menu later.

**Pull request merged**